### DISCUSSION TITLE
Automatically assosciate all scripts to the same data source

### DISCUSSION THREAD

**Discussion by chinwobble** [dbeaver/dbeaver#11505] [Q&A] [answered]
I have hundreds of sql scripts that all act on the same database.
I imported them into a dbeaver project.
I want them all to be associated with the one connection I setup.

**Describe the solution you'd like**
Add "associate to data source" with the right click menu of a folder.
This will allow all scripts in a folder to be associated.

**Describe alternatives you've considered**
Manually assosciating all sql scripts.

**Comment by kseniiaguzeeva** [comment, marked as answer]
Please select your scripts in the folder and press **Assosiate with data source** in the context menu.
![dbeaver_FMCpKcqYUM](https://user-images.githubusercontent.com/51405061/109164061-0b610800-778b-11eb-9857-b0666c656121.png)
